docs/rfcs/managed.md

```diff
@@ -0,0 +1,150 @@
+# Managed CRD
+
+**Status:** *Draft*
+
+**Author:** Antoine Toulme (@atoulme)
+
+**Date:** 2025-03-20
+
+## Objective
+
+Condense all best practices and deployment patterns of OpenTelemetry components into a single custom resource
+that covers the feature set of the OpenTelemetry Helm chart.
```

Comment:
I would rephrase this sentence, I am not sure what the helm chart supports and on top of that there are multiple helm charts https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts

Reply:
```suggestion
that covers the use cases fulfilled by the OpenTelemetry helm charts.
```

Reply:
done :)